src/main/resources/templates/client/show_clients.ftl

```diff
@@ -46,13 +43,16 @@
             <td>${client.lastName}</td>
             <td>${client.email}</td>
             <th scope="col">
-            <span style="margin-left: 27%;">
-            <form style='display:inline; width: 50px ' action="/clients/update/${client.id}" method="get">
-                <button class="btn btn-outline-success" type="submit">Edit</button>
-            </form>
-            <form style='display:inline; width: 50px' action="/clients/delete/${client.id}" method="post" >
-                <button class="btn btn-outline-danger" type="submit">Delete</button>
-            </form>
+            <span style="margin-left: 13%;">
+                <form style='display:inline; width: 50px' action="/orders/product/${client.id}" method="get">
```

Comment:
Do not use form to perform get request. Use regular <a> instead

Reply:
Okay, thanks